src/app/app.component.ts

```diff
@@ -1,10 +1,20 @@
 import { Component } from '@angular/core';
+import { Person } from './classes/person';
 
 @Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+ selector: 'app-root',
+ templateUrl: './app.component.html',
+ styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'app';
+ title = 'app';
+ heading = 'Person Details';
+ person: any;
```

Comment:
person: any = {
  name: {},
  gender: ''
};

^ is where you would start from